Resfeber/Model Controller/ProfileController.swift

```diff
@@ -142,55 +139,55 @@ class ProfileController {
             }
             return
         }
-        
+
         let requestURL = baseURL
             .appendingPathComponent("profiles")
             .appendingPathComponent(userID)
         var request = URLRequest(url: requestURL)
-        
+
         request.addValue("Bearer \(oktaCredentials.idToken)", forHTTPHeaderField: "Authorization")
-        
-        let dataTask = URLSession.shared.dataTask(with: request) { (data, response, error) in
-            
+
+        let dataTask = URLSession.shared.dataTask(with: request) { data, response, error in
+
             var fetchedProfile: Profile?
-            
+
             defer {
                 DispatchQueue.main.async {
                     completion(fetchedProfile)
                 }
             }
-            
+
             if let error = error {
                 NSLog("Error getting all profiles: \(error)")
             }
-            
+
             if let response = response as? HTTPURLResponse,
-                response.statusCode != 200 {
+               response.statusCode != 200
+            {
```

Comment:
I think the opening brace should remain on the preceding line to keep consistent with how the rest of the code is formatted.